pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -1691,7 +1691,7 @@ func TestUpdatePodSandboxResources(t *testing.T) {
 	assert.Len(t, fakeContainers, 1)
 
 	ctx := context.Background()
-	_, err := m.getPodContainerStatuses(ctx, pod.UID, pod.Name, pod.Namespace)
+	_, _, err := m.getPodContainerStatuses(ctx, pod.UID, pod.Name, pod.Namespace, "")
```

Comment:
rebased to fix the conflict due to the function signature change.

@SergeyKanzhelev @toVersus
Can you re-lgtm on this?

Reply:
I'm not a member of kubernetes org yet, so I don't have permission to use /lgtm.

@SergeyKanzhelev @matthyx 
Could one of you handle it instead?